Swift/Tests/VectorSearchTest.swift

```diff
@@ -701,28 +688,12 @@ class VectorSearchTest: CBLTestCase {
     ///       doesn’t exist in the log.
     ///     9. Reset the custom logger.
     func testCreateVectorIndexWithNoneEncoding() throws {
-        let collection = try db.collection(name: "words")!
         var config = VectorIndexConfiguration(expression: "vector", dimensions: 300, centroids: 8)
         config.encoding = .none
-        try collection.createIndex(withName: "words_index", config: config)
-        
-        let names = try collection.indexes()
-        XCTAssert(names.contains("words_index"))
-        
-        // Query:
-        let sql = "select meta().id, word from _default.words where vector_match(words_index, $vector, 20)"
-        let parameters = Parameters()
-        parameters.setValue(dinnerVector, forName: "vector")
-        
-        let q = try self.db.createQuery(sql)
-        q.parameters = parameters
+        try createWordsIndex(config: config)
         
-        let explain = try q.explain() as NSString
-        XCTAssertNotEqual(explain.range(of: "SCAN kv_.words:vector:words_index").location, NSNotFound)
-        
-        let rs = try q.execute()
+        let rs = try executeWordsQuery(limit: 20)
         XCTAssertEqual(rs.allResults().count, 20)
-        XCTAssert(checkIndexWasTrained())
     }
     
     /// FAILED : https://issues.couchbase.com/browse/CBL-5538
```

Comment:
Can we re-enable 12 bits?

Reply:
Yes. Can you do that when go over all of the failed tests?